.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Question
+labels: question
+about: Ask a general question related to OpenSSL development
+---
+
+<!--
+Thank you for your interest in OpenSSL. If this is your first question,
+please take the time to read the following lines before posting it.
+
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
```

Comment:
Hmmm, users are not encouraged to raise a question about how to use OpenSSL on Github?

Reply:
Well, the formulation was taken from the other two templates and it used to be the policy up to now (see also this commit message). I wouldn't object to changing it or removing it, but that would at least require some discussion by the @openssl/omc members.

Reply:
Perhaps this is also a good topic on the coming f2f meeting ;-)

Reply:
We actually did discuss this a bit on the last f2f, and didn't get to a complete agreement, but our behaviour *has* changed since to start answering questions here as well (we used to tell people to go to openssl-users when they were asking question before, I don't think anyone of us is doing that any more...  pragmatically speaking, it makes sense).

Also, having this text in an issue variant that's *all* about asking questions is a bit absurd, it kinda defeats the purpose, doesn't it?

Reply:
> Also, having this text in an issue variant that's _all_ about asking questions is a bit absurd, it kinda defeats the purpose, doesn't it?

Well, the idea behind it was: the question template is for those who _don't_ read this message... ;-)

Reply:
But seriously: I'll be happy to remove or change the text. But then we should do it in all three templates.

Reply:
I have an idea how this paragraph could be reformulated...